src/Exceptions/HttpInsufficientStorage.php

```diff
@@ -8,4 +8,3 @@ class HttpInsufficientStorage extends HttpException {
     public const DESCRIPTION = "Means the method could not be performed on the resource because the server is unable to store the representation needed to successfully complete the request.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/507";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces